codebase/codebase.go

```diff
@@ -37,22 +40,41 @@ func (c *CodebaseContent) ToMap() map[string]interface{} {
 	res[BranchKey] = c.Branch
 	res[FileNameKey] = c.FileName
 	res[LineNumberKey] = c.LineNumber
+	res[CodebaseIDKey] = c.CodebaseID
 	return res
 }
 
+// validateURL makes sure Repo is valid GIT URL
+func validateURL(repo string) (string, bool) {
+	validPrefixes := []string{"https://github.com/", "git@github.com:"}
+	if strings.HasPrefix(repo, validPrefixes[0]) || strings.HasPrefix(repo, validPrefixes[1]) {
+		if strings.HasSuffix(repo, ".git") == false {
+			return repo + ".git", true
```

Comment:
@pranavgore09 Don't be blocked by my comment. I believe we don't get that at this moment from the analyzer. So. move forward :)